app/helpers/visit_helper.rb

```diff
@@ -46,7 +46,7 @@ def prison_slot_anomalies
   end
 
   def prison_address(glue='<br>')
-    prison_data['address'].join(glue).html_safe
+    safe_join(prison_data['address'], glue.html_safe)
```

Comment:
@threedaymonk I couldn't see another way than what @EGiataganas has done here to make the `glue` render as actual markup… Don't suppose you know of one?